packages/@okta/vuepress-site/docs/guides/sharing-cert/overview/index.md

```diff
@@ -2,25 +2,20 @@
 title: Overview
 ---
 
-[Application key credential objects](/docs/reference/api/apps/#application-key-credential-model)
-contain an opaque key ID (`kid`) and corresponding certificate.
-To use the [API](/docs/reference/api/apps/#clone-application-key-credential)
-to share application key credentials between apps, you need to create and use a new credential in one app,
-then share and update the credential in another app.
+[Application Key Credential objects](/docs/reference/api/apps/#application-key-credential-model)
+contain an opaque key ID (`kid`) and corresponding certificate. To use the [API](/docs/reference/api/apps/#clone-application-key-credential) to share application key credentials between apps, you need to create and use a new credential in one app, and then share and update the credential in another app.
```

Comment:
Suggested re-write:
To **share application key credentials between apps, you need to [clone an application key credential](/docs/reference/api/apps/#clone-application-key-credential). You** create and use a new credential in one app, and then share and update the credential in another app.